bom/pom.xml

```diff
@@ -156,12 +161,12 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.cxf</groupId>
-                <artifactId>cxf-rt-management</artifactId>
+                <artifactId>cxf-rt-javascript-tests</artifactId>
```

Comment:
The tests artifacts (like `cxf-rt-javascript-tests`) should very likely not be there

Reply:
Removed cxf-rt-javascript-tests